bin/typescript-axios-petstore-all.sh

```diff
@@ -4,5 +4,6 @@
 ./bin/typescript-axios-petstore-with-npm-version.sh
 ./bin/typescript-axios-petstore-with-npm-version-and-separate-models-and-api.sh
 ./bin/typescript-axios-petstore-with-complex-headers.sh
+./bin/typescript-axios-petstore-with-single-request-parameters.sh
```

Comment:
please also add a windows version to the corresponding `bin/windows/typescript-axios-petstore-all.bat`